git.hooks code/pre-push

```diff
@@ -5,6 +5,8 @@ import sys
 from subprocess import Popen, PIPE
 import py_compile
 
+Popen('chmod +x pre-push', shell=True, stdout=PIPE)
```

Comment:
The same as above

Reply:
this string is absent in master branch, I deleted this file from current branch to save the file in master after merge